.github/workflows/preview-comment.yaml

```diff
@@ -11,6 +11,10 @@ on:
         description: 'SHA of the commit associated with this Pull Request'
         required: true
         type: string
+      site_name: 
+        description: 'Name of the website in comment'
+        required: false
+        default: "The Cookbook"
```

Comment:
I think this needs
```
type: string
```
unless that is interpreted implicitly from the default value?

Reply:
Sounds good to me (but I am also not very knowledgeable in this regard hehe).